### PR TITLE
Register ILogger<T> -> Logger<T> in default services

### PR DESCRIPTION
**Pull request by DamianEdwards**
See https://github.com/aspnet/Logging/pull/96

### CODE REVIEW

**Comment by davidfowl** [comment]
Omg this extension method is in Hosting?

**Comment by DamianEdwards** [comment]
@davidfowl yeah I had that discussion with @loudej, it's there 'cos the ANIs are conditional on it targeting ASPNET50, blah blah.

**Comment by davidfowl** [comment]
:shipit:

File: src/Microsoft.AspNet.Hosting/HostingServicesCollectionExtensions.cs [hunk 1]
```diff
@@ -17,6 +17,7 @@ public static IServiceCollection AddLogging(this IServiceCollection services, IC
         {
             var describe = new ServiceDescriber(config);
             services.TryAdd(describe.Singleton<ILoggerFactory, LoggerFactory>());
+            services.TryAdd(describe.Singleton(typeof(ILogger<>), typeof(Logger<>)));
```
Comment (DamianEdwards) [inline src/Microsoft.AspNet.Hosting/HostingServicesCollectionExtensions.cs]: @davidfowl are you sure this should be a singleton? `ILogger` and `Logger` in particular don't appear to be thread safe.

**Comment by DamianEdwards** [comment]
Merged in eba2808109f86b034466051d0abccf2d01cf81fa

**Pull request closed without merging**